src/main/groovy/io/openliberty/tools/gradle/utils/ServerUtils.groovy

```diff
@@ -348,17 +355,29 @@ class ServerUtils {
                     }
                 }
             }
-            
+
             // Wait a bit for processes to be killed
             Thread.sleep(1000)
-            
+
         } catch (Exception e) {
             logger.warn("Error during force kill: " + e.getMessage())
         } finally {
-            ProcessUtils.closeQuietly(reader, logger, "reader")
             if (findProcess != null) {
                 ProcessUtils.drainAndCloseProcessStream(findProcess, true, logger)
             }
         }
     }
+
+    static void cleanupForceStoppedMarker(File serverDir, Logger logger) {
+        File forceStoppedMarker = new File(serverDir, ".liberty_plugin_force_stopped")
+        if (forceStoppedMarker.exists()) {
+            logger.debug("Removing liberty_plugin_force_stopped marker file from previous server session")
+            boolean deleted = forceStoppedMarker.delete()
+            if (!deleted) {
```

Comment:
No need to have separate lines for this. Just change to `if (!forceStoppedMarker.delete())`.